plonky2x/benches/mux_bench.rs

```diff
@@ -0,0 +1,37 @@
+#![feature(generic_const_exprs)]
+extern crate plonky2x;
+
+use criterion::{criterion_group, criterion_main, Criterion, BenchmarkId};
+use ethers::types::U256;
+use plonky2::field::types::Field;
+use plonky2x::{frontend::{builder::CircuitBuilder, vars::U256Variable}, prelude::{Variable, GoldilocksField, CircuitVariable}};
+
+fn benchmark_select_index(c: &mut Criterion) {
+    let mut group = c.benchmark_group("mux");
+    group.sample_size(10);
+
+    type F = GoldilocksField;
+    const D: usize = 2;
+    let mut builder = CircuitBuilder::<F, D>::new();
+    let selector = builder.constant::<Variable>(F::from_canonical_usize(42)); // Example selector value
+    let inputs: Vec<u64> = (1..=74851).collect();
+
+    let input_variables = inputs
+            .iter()
```

Comment:
This benchmark is somewhat unnecessary to compare the two implementations, you can measure how expensive it is the run the circuit by printing the number of gates.

I think there is a method like builder.api.nb_gates()

Reply:
As a result, I think we can remove this file

It's also not a fair comparison due to the way Plonky2 pads the plonk table to the next power of two

Reply:
this issue arise when i try to add gate print for diff impl for mux but need to back and forth, and also feel it's not very systematic.

why this is not fair? actually even with this number, random_access is fast, but the issue is current impl of select_with_random_access overflow with lot of inputs, do u mind help take a look?
https://github.com/succinctlabs/succinctx/pull/130/commits/b239731e2ab07a65be83df0d32e5cb83fa4cb038#diff-ba6371572665a5385874a1d7bf91cead2351f50f015206f1e33cbc2643a16036R128